src/db/dal/user.ts

```diff
@@ -28,10 +134,42 @@ export const searchUsers = async (pageSize: number, pageIndex: number) => {
     };
 };
 
-export const getUserById = async (keycloak_id: string): Promise<IUserOuput> => {
+export const getProfileImageUploadPresignedUrl = async (keycloak_id: string) => {
```

Comment:
- just a reminder that this has a security/privacy issue. At the very least, meta tags are not removed.
-  I think the naming `getProfileImageUploadPresignedUrl` implies that we `get` something but actually I think we `get` and or `put` something

Reply:
- De quel probleme de securité parles tu ? Indique moi un lien qui justifie ta source d'inquietude
- C'est utilisé pour avoir quelque chose, le get semble approprié. Encore une fois c'est un merge de include le but ici est pas de remettre en question tout le code existant mais limité les risques de breaking changes.
  ```const result = await getProfileImageUploadPresignedUrl(keycloak_id);
        res.status(StatusCodes.OK).send(result);```